starter-code/src/app/contact-list/contact-list.component.ts

```diff
@@ -1,24 +1,33 @@
-import { Component, OnInit } from '@angular/core';
-import contactList from '../contacts';
+import { Component, OnInit } from "@angular/core";
+import contactList from "../contacts";
 
 @Component({
-  selector: 'app-contact-list',
-  templateUrl: './contact-list.component.html',
-  styleUrls: ['./contact-list.component.css']
+  selector: "app-contact-list",
+  templateUrl: "./contact-list.component.html",
+  styleUrls: ["./contact-list.component.css"]
 })
 export class ContactListComponent implements OnInit {
   contacts: Object[];
-  newContact: Object = {};
+  newContact: Object = new Contact();
```

Comment:
So if you created a `Contact` class, why not declare every instance of it as type `Contact`?

`newContact: Contact = new Contact();`

Alternatively, you can choose to not declare a type here at all, since when you use the `new...` statement it's pretty obvious which class it belongs to, so simply:
`newContact = new Contact();`